sources/params.move

```diff
@@ -0,0 +1,72 @@
+module bitcoin_spv::params;
+
+public struct Params has store{
+    power_limit: u256,
+    blocks_pre_retarget: u64,
+    /// time in seconds when we update the target
+    target_timespan: u64,
+    pow_no_retargeting: bool,
+    reduce_min_difficulty: bool, // for Bitcoin testnet
+    min_diff_reduction_time: u32,  // time in seconds
+}
+
+// default params for bitcoin mainnet
+public fun mainnet(): Params {
+    return Params {
+        power_limit: 0x00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff,
+        blocks_pre_retarget: 2016,
+        target_timespan: 2016 * 60 * 10, // ~ 2 weeks.
+        pow_no_retargeting: false,
+        reduce_min_difficulty: false,
+        min_diff_reduction_time: 0,
+    }
+}
+
+// default params for bitcoin testnet
+public fun testnet(): Params {
+    return Params {
+        power_limit: 0x00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff,
+        blocks_pre_retarget: 2016,
+        target_timespan: 2016 * 60 * 10, // ~ 2 weeks.
+        pow_no_retargeting: false,
+        reduce_min_difficulty: true,
+        min_diff_reduction_time: 20 * 60, // 20 minutes
+    }
+}
+
+// default params for bitcoin regtest
+// https://github.com/bitcoin/bitcoin/blob/v28.1/src/kernel/chainparams.cpp#L523
+public fun regtest(): Params {
+    return Params {
```

Comment:
You can remove return, move have expression like rust

Reply:
thanks, updated